src/components/ChataTable/ChataTable.scss

```diff
@@ -103,6 +103,11 @@
   height: 7px;
   transition: all 0.3s ease;
 }
+.react-autoql-table.tabulator .tabulator-tableHolder{
+  min-height: calc(100% - 40px)  !important;
+  height: calc(100% - 40px) !important;
+  max-height: calc(100% - 40px) !important;
```

Comment:
when you have `height` defined, you shouldn't need to define `min-height` and `max-height` as well

Reply:
> when you have `height` defined, you shouldn't need to define `min-height` and `max-height` as well

oh, thank you. I can clean it.